plat/rockchip/rk3368/drivers/pmu/pmu_cpus_on.S

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2014, Fuzhou Rockchip Electronics Co., Ltd
+ * Author: Tony Xie <tony.xie@rock-chips.com>
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms and conditions of the GNU General Public License,
+ * version 2, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ */
+
+#include <arch.h>
+#include <asm_macros.S>
+#include <rk3368_def.h>
+#include <pmu.h>
+
+	.globl pmu_cpuson_entrypoint_start
+	.globl pmu_cpuson_entrypoint_end
+
+func pmu_cpuson_entrypoint
+pmu_cpuson_entrypoint_start:
```

Comment:
Sorry, update it.
1、The cpu reset or warm boot address in this soc needs aligned 64K bytes. so it can not be pointed to the entrypoit function in psci freely.
2、If the system resumes from system suspend，the boot code needs to be executed in pmusram in which we have the ddr initiated because of The DDR Controller power down.
So we program this position-independent code.

Maybe, let me know if you have some suggestions or ideas. :)